includes/repositories/EUtilsAssemblyRepository.php

```diff
@@ -32,9 +38,139 @@ class EUtilsAssemblyRepository extends EUtilsRepository {
    */
   public function create($data) {
 
```

Comment:
So required_fields are not enforced by default. We need to call this function:

```suggestion
$this->validateFields($data);
```

Reply:
i think this will fail because i mistake the required fields array for the chado base record.  I guess we should have both: the base record fields that im assembling, and then required fields that will go through the validator, which just makes sure that all the right keys are there passed via the XML.